index.js

```diff
@@ -49,7 +49,8 @@ module.exports = function (options) {
     }).then(function(file) {
       cb(null, file);
     }).catch(function(err) {
-      // Convert the keys so PluginError can read them
+      // Add the keys so PluginError can read them
+      err.formattedMessage = less.engine.formatError(err, { color: true });      
```

Comment:
why "formattedMessage"? This won't be displayed anywhere - this should be `err.message` if you want it to show up when logged